.gitignore

```diff
@@ -19,3 +19,7 @@ coverage
 
 # bundled files
 lib/bundle.js
+
+package-lock.json
+
+front-end-testing/test/*
```

Comment:
@ZooeyMiller got it, but do we _want_ to exclude the "solution" from the repo at the expense of people's understanding? (_never assume you will be able to answer the person's "**I'm confused**" questions in-person... the answer needs to be **somewhere** if they get stuck._)